solutions/observability/apps/upgrade-self-installation-of-apm-server-standalone-to-8x.md

```diff
@@ -49,6 +50,8 @@ This upgrade guide is for the standalone method of running APM Server. Only use
 
     Additional details are available in [start the APM Server](apm-server-binary.md#apm-server-starting).
 
-5. **(Optional) Upgrade to the APM integration**
+5. When upgrading from 7.x to 9.0 if you reindex 7.x indices, you need to either add [ILM privileges](https://www.elastic.co/guide/en/apm/guide/7.17/privileges-to-setup-beats.html#_set_up_ilm) for `reindexed-v*-apm*` indices or set the indices to `readonly`.
```

Comment:
Similar to above, upgrade from 7.x to 9.0 is not supported, this needs to discuss upgrade 8.18 to 9.0 if indices from 7.x still exist.